src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -757,9 +755,18 @@ public class SimpleEvents {
 				.description("Called when a player drags an item in their cursor across the inventory.")
 				.examples(
 						"on inventory drag:",
-						"\tif player's current inventory is {_gui}:",
-						"\t\tsend \"You can't drag your items here!\" to player",
-						"\t\tcancel event"
+							"\tif player's current inventory is {inventories::custom-gui}:",
+							"\t\tsend \"You can't drag your items here!\" to player",
+							"\t\tcancel event"
+				)
+				.since("INSERT VERSION");
+
+		Skript.registerEvent("Server Broadcast", SimpleEvent.class, BroadcastMessageEvent.class, "[server] broadcast")
+				.description("Called when the server broadcasts messages.")
```

Comment:
```suggestion
				.description("Called when the server broadcasts messages. Does not get called when using Skript's <a href='effects.html#EffBroadcast'>broadcast</a> effect.")
```

Reply:
Will #5979 be merged in a different release? We should make sure we don't accidentally end up documenting the opposite of the actual behaviour

Reply:
That PR does not have a targeted release. Let me know if it will be released altogether.